test/Concurrency/experimental_feature_strictconcurrency.swift

```diff
@@ -20,3 +20,33 @@ struct Test2: TestProtocol { // expected-warning{{conformance of 'C2' to 'Sendab
   // expected-note@-1{{in associated type 'Self.Value' (inferred as 'C2')}}
   typealias Value = C2
 }
+
+@globalActor
+actor TestGlobalActor {
+  static var shared = TestGlobalActor()
+}
+
+@TestGlobalActor
+var mutableIsolatedGlobal = 1
+
+var mutableNonisolatedGlobal = 1 // expected-error{{var 'mutableNonisolatedGlobal' is not concurrency-safe because it is involves shared mutable state}}
```

Comment:
I'd nitpick the error a bit to be honest "involves" is not quite right here.

- maybe `"variable 'mutableNonisolatedGlobal' because it is global shared mutable state"`
- consider "variable" vs "var" I don't think we use short names like that in messages much
- consider adding a note that can explain what people can do about it?
  - maybe "note: isolate this property using a global actor, or convert it to a constant"?

Reply:
The `var` part is generated via `%kind0` however otherwise making these changes, thanks for the improvement suggestions.